Sentinel/Classes/DatabaseImportExport/Internal/DatabaseFileManager.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  DatabaseFileManager.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 12.03.2025..
+//
+
+import Foundation
+
+public enum DatabaseFileError: Error {
+    case importFailed
+    case exportFailed
+}
+
+final class DatabaseFileManager {
+
+    // MARK: - Private properties
+
+    private let fileManager: FileManager
+    private let databaseFilePath: String
+
+    private lazy var documentDirectoryUrl: URL? = {
+        fileManager.urls(for: .documentDirectory, in: .userDomainMask).first
+    }()
+
+    private lazy var exportDirectoryUrl: URL? = {
+        documentDirectoryUrl?.appendingPathComponent("export")
+    }()
+
+    private lazy var exportArchiveUrl: URL? = {
+        exportDirectoryUrl?.appendingPathComponent("export.zip")
+    }()
+
+    private lazy var databaseURL: URL? = {
+        guard let documentDirectoryUrl else { return nil }
+        return URL(fileURLWithPath: "\(documentDirectoryUrl.path)/\(databaseFilePath)")
+    }()
+
+    // MARK: - Init
+
+    init(fileManager: FileManager = .default, databaseFilePath: String) {
+        self.fileManager = fileManager
+        self.databaseFilePath = databaseFilePath
+    }
+
+    func importDatabase(from url: URL) throws {
+        defer { url.stopAccessingSecurityScopedResource() }
+        guard
+            let databaseURL,
+            url.startAccessingSecurityScopedResource()
+        else { return }
+        deleteDirectoryIfExists(atPath: databaseURL.path)
+
+        _ = try fileManager.copyItem(at: url, to: databaseURL)
+    }
+
+    func exportDatabase() -> URL? {
```

Comment:
Regarding the export, should we provide option to export the database itself (not an archive) and the archived file? I'm just asking due to the consistency, feel free to argument me on those two comments 😄👍